Tests/Fixtures/Generated/XCAssets/swift3/all-customName.swift

```diff
@@ -2,11 +2,16 @@
 // Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
 
 #if os(OSX)
-  import AppKit.NSImage
+  import AppKit
   internal typealias XCTColor = NSColor
   internal typealias XCTImage = NSImage
-#elseif os(iOS) || os(tvOS) || os(watchOS)
-  import UIKit.UIImage
+#elseif os(iOS)
+  import ARKit
```

Comment:
That... might be a bit easier said than done. I'll look into it.

Reply:
Might be worth making the parser do the counting (like exposing the number of resources for each type in the context) so that it'd be easier to then test `{% if resourceCount.arresourcegroup > 0 %}` in the template — instead of having the template loop on the list of all resources and checking the type of each one etc all in the template